qa/L0_large_payload/large_payload_test.py

```diff
@@ -157,6 +157,7 @@ def test_onnx(self):
         for client in self._clients:
             model_name = tu.get_zero_model_name("onnx_nobatch", 1,
                                                 self._data_type)
+            self._test_helper(client, model_name)
```

Comment:
So onnx was not running the large payload test!?!

Reply:
Yes it was extremely odd. I did not see a comment as to why it was not there so I added it. 
I was able to trace back the removal to https://github.com/triton-inference-server/server/pull/2207